pages/index.js

```diff
@@ -1,16 +1,40 @@
 import Head from 'next/head'
 import Image from 'next/image'
 import styles from '../styles/Home.module.css'
-import Coming from '@components/ComingSoon/comingsoon'
+import Header from '@components/ComingSoon/Header'
+
 export default function Home() {
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>DHRUVA - IIITK</title>
-        <meta name="description" content="Dhruva IIITK Technical Fest" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-        <Coming/>
-    </div>
-  )
+    return (
+        <div className={styles.container}>
+            <Head>
+                <title>DHRUVA - IIITK</title>
+                <meta
+                    name="description"
+                    content="Dhruva IIITK Technical Fest"
+                />
+                <link rel="icon" href="/favicon.ico" />
+                <link rel="preconnect" href="https://fonts.googleapis.com" />
+                <link
+                    rel="preconnect"
+                    href="https://fonts.gstatic.com"
+                    crossorigin
```

Comment:
Are you sure it is crossorigin.. and not crossOrigin
Because following error occurred while deploying:
Error: Unknown property 'crossorigin' found, use 'crossOrigin' instead  react/no-unknown-property